spec/controllers/teams/users_controller_spec.rb

```diff
@@ -84,6 +96,19 @@
         end
     end
 
+    context 'when user is a team member and user to delete is team member' do
+      before do
+          team.users << user_to_delete
+          team.users << user
+          delete :destroy, params: { team_id: team.id, id: user_to_delete.id }
+      end
+
+      it 'user no longer in team' do
```

Comment:
Beautiful spec! Only thing I suggest you change is to move it all into the same context as the context starting at line 87. The nice thing about contexts is that you can share them between tests when you're setting up a similar state!